recipes/derekstride-tree-sitter-sql/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class DerekstrideTreeSitterSqlConan(ConanFile):
+    name = "derekstride-tree-sitter-sql"
```

Comment:
Everything looks good, but I have a question here. Why are we adding the authors name inside the recipe name?
Wouldn't be more convenient to call it `tree-sitter-sql` as it is called in cargo and pip https://github.com/DerekStride/tree-sitter-sql#using-cargo?
Also to be more homogeneous with existing tree-sitter recipes.

There are other projects, TJ one https://github.com/tjdevries/tree-sitter-sql and https://github.com/m-novikov/tree-sitter-sql
First seems to be abandoned and the other haven't been updated in a while. 
So I will suggest to rename to `tree-sitter-sql` and in the future we may consider adding other providers.
What do you think? It may be a personal preference.

Reply:
I agree in renaming it to `tree-sitter-sql`. The project seems be consolidated over other package managers and most active between their homonyms.

Reply:
@perseoGI @uilianries 
Thank you for your advise!
I will rename this recipe.

I had used to use m-novikov's `tree-sitter-sql` in previous, which was the most advanced implementation at the time.
Since m-novikov's tree-sitter-sql is not that old, I wondered if I should give such a generic name as `tree-sitter-sql` only to derekstride's.

Reply:
fixed.

Please review it again.

Reply:
Thank you @toge san! :smile: